src/cluecode/finder_data.py

```diff
@@ -49,7 +51,6 @@ def set_from_text(text):
     example.com
     example.net
     example.org
-    test.com
```

Comment:
Firstly, thanks for the (very) thorough review, will look into everything you've suggested! 
I removed it, as it was being checked for using the `exact_match` test, but now I see cases (thanks to you!) where it could fail. Will work on it!